src/test/java/core/basesyntax/MainTest.java

```diff
@@ -0,0 +1,112 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.servise.FileReaderUtility;
+import core.basesyntax.servise.FruitTransactionService;
+import core.basesyntax.servise.impl.FileReaderUtilityImp;
+import core.basesyntax.servise.impl.FruitTransactionServiceImp;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class MainTest {
```

Comment:
Do not test all logic in one class
The structure of the tests must correspond to the structure of the project
Each service handler strategy should have its own test class